### ISSUE TITLE
Run Info View - "Started At" is picking wrong time.

### ISSUE THREAD

**Issue by vishwathma** [anshooarora/extentreports#28]
Please have a look at the screenshots attached.
![wrong time](https://cloud.githubusercontent.com/assets/13534392/8929254/a6179c8e-3541-11e5-854e-797731321621.jpg)
![wrong time - ii](https://cloud.githubusercontent.com/assets/13534392/8929255/a617e770-3541-11e5-82cd-77fd6eff55a0.jpg)

**Comment by anshooarora** [comment]
I am unable to reproduce this.  Please share your code.

Are you starting the ExtentTest when the test actually starts or are you using it as a listener?  If you use it as a listener then you will face this issue..

**Comment by vishwathma** [comment]
I have sent a mail, with the attached code. Please have a look.

**Comment by anshooarora** [comment]
Vishi

In the report you have emailed, the started and ended times from RunInfo view are:

2015-07-30 10:42:41

Which is why the difference is 0.  

If you are using this via listener, then you will have to manually set start and end times using:

```
test.getTest().startedTime = startedTimeFromTestNG;
test.getTest().endedTime = endedTimeFromTestNG;
```

See this thread for more info:  https://github.com/anshooarora/extentreports/issues/32

**Comment by anshooarora** [comment]
Vishi - you can use the new ExtentReporterNG code as-is and it will also report start and end-times:

http://relevantcodes.com/testng-listener-using-extentreports/

**Comment by anshooarora** [comment]
Issue fixed in ad6b91d. Closing ticket.
